frontend/utils/useFetch.tsx

```diff
@@ -0,0 +1,20 @@
+import { useEffect, useState } from "react";
+import axios from "axios";
+const useFetch = (options:{}) =>{
+    const [json, setJson] = useState("");
+
+    function call(option:{}){
+          axios.request(option).then(function (response:any){
+            setJson(response.data);
```

Comment:
Make sure to type the response data using custom type

Reply:
But this function returns just json file of any kind. We will choose later what type is it based on context

Reply:
My comment refers to the line with setJson(response.data);